tester_app/app/assets/stylesheets/_bootstrap-custom.scss

```diff
@@ -0,0 +1,53 @@
+// Core variables and mixins
```

Comment:
// This file is used instead of 
// @import 'boostrap';
// To allow for customization of what parts of bootstrap are imported.